backend/frontend/urls.py

```diff
@@ -39,6 +39,11 @@
         views.action_start_card,
         name="action_start_card",
     ),
+    path(
+        "cards/<int:card_id>/details",
```

Comment:
We need to navigate to projects even when they don't have cards. `projects/<int:card_id>` would be better. The `/details` at the end seems unnecessary.